### PR TITLE
W-9871892: [Accessibility] The “Trailhead” logo image does not contain any proper information or meaningful text

### PR DESCRIPTION
**Pull request by JoseRossiSF**
# Work Item
[W-9871892](https://gus.lightning.force.com/a07EE000005MMYbYAO)

# Changes
Added "alt" attribute for Trailhead Logo for gseuTrailhead LWC to fix accessibility issue.

### CODE REVIEW

File: force-app/main/default/lwc/gseuTrailhead/gseuTrailhead.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
     <lightning-card>
         <lightning-layout horizontal-align="space" multiple-rows="true">
             <lightning-layout-item size="12" class="slds-text-align_center">
-                <img class="" src={imgUrl} alt=""/>
+                <img class="" src={imgUrl} alt="Trailhead Logo"/>
```
Comment (psadie-sfdo) [inline force-app/main/default/lwc/gseuTrailhead/gseuTrailhead.html:5]: Need a custom label here. Please create one for "Trailhead Logo" if it doesn't already exist. Thanks!

**Review by psadie-sfdo — CHANGES_REQUESTED** [review]

File: force-app/main/default/lwc/gseuTrailhead/gseuTrailhead.html (hunk 1 above)
Comment (JoseRossiSF) [reply force-app/main/default/lwc/gseuTrailhead/gseuTrailhead.html]: Agreed. Change done. Thank you, Prince

**Review by psadie-sfdo — APPROVED** [review]
Looks good!

**Review by dcaballeroSf — APPROVED** [review]
LGTM

**Review by mcasco — APPROVED** [review]
LGTM

**Pull request merged**